manifests/config/passenger.pp

```diff
@@ -175,6 +180,67 @@
         custom_fragment         => template('foreman/_assets.conf.erb', 'foreman/_ssl_virt_host_include.erb',
                                             'foreman/_suburi.conf.erb'),
       }
+
+      $pki_dir = '/etc/pki/foreman'
```

Comment:
`/etc/pki` is something that only exists on RH-based distros. I'm not sure what would be the best place for this that's generic for all distros.